internal/connector/common/resources_common.go

```diff
@@ -60,3 +60,11 @@ func GetManagementEmbedded(apiExecuteFunc func() (*management.EntityArray, *http
 
 	return embedded, nil
 }
+
+func GenerateCommentInformation(data map[string]string) string {
+	commentInformation := "\n"
```

Comment:
Should this be added even if `data` is empty?

Reply:
It would just add an empty line in between import blocks, which allows for easy readability anyway. So I think this is good as is.